packages/orchestration/src/utils/orchestrationAccount.js

```diff
@@ -13,6 +13,17 @@ import {
 
 const { Vow$ } = NetworkShape; // TODO #9611
 
+/**
+ * Mark a possibly incomplete object as complete by removing the `incomplete`
+ * property.
```

Comment:
Is there no way to determine from the object's properties whether it's complete? This is pretty janky.